apis/r/DESCRIPTION

```diff
@@ -1,9 +1,11 @@
 Package: tiledbsoma
 Type: Package
-Title: TileDB SOMA
+Title: 'TileDB' Stack of Matrices, Annotated ('SOMA')
 Description: Interface for working with 'TileDB'-based Stack of Matrices,
     Annotated ('SOMA'): an open data model for representing annotated matrices,
-    like those commonly used for single cell data analysis.
+    like those commonly used for single cell data analysis. It is documented at
+    <https://github.com/single-cell-data>; a formal specification available is at
+    <https://github.com/single-cell-data/SOMA/blob/main/abstract_specification.md>.
```

Comment:
Good call including the spec.

Reply:
Yes -- word of mouth is that 'first pass' for new packages often asks for references to published work underlying it etc.  So this is aiming to address that question.